HDF5Examples/CMakeLists.txt

```diff
@@ -93,27 +93,51 @@ if (H5_HAVE_PARALLEL)
   INCLUDE_DIRECTORIES (${MPI_C_INCLUDE_DIRS})
 endif ()
 
+# Determine if a threading package is available on this system
+option (HDF5_ENABLE_THREADS "Enable thread support" ON)
+set (THREADS_PREFER_PTHREAD_FLAG ON)
+find_package (Threads)
+if (Threads_FOUND)
+  set (H5_HAVE_THREADS 1)
+  set (CMAKE_REQUIRED_LIBRARIES ${CMAKE_THREAD_LIBS_INIT})
+
+  # Determine which threading package to use
+  #CHECK_INCLUDE_FILE("threads.h" HAVE_THREADS_H)
```

Comment:
Also looks like this maybe wasn't meant to be commented out.

Reply:
Same (supposed to be commented out until we enable C11 support). I'll add a comment explaining it here too.